Source/Forms/Gen4/Wild4.cpp

```diff
@@ -296,12 +297,32 @@ void Wild4::search()
     searcherModel->clearModel();
     searcherModel->setMethod(method);
```

Comment:
These need to be moved after the 31IV check

Reply:
done